spyder/api/widgets/menus.py

```diff
@@ -83,7 +116,7 @@ def clear_actions(self):
         self._sections = []
         self._actions = []
         self._actions_map = {}
-        self.unintroduced_actions = {}
+        self._unintroduced_actions = {}
```

Comment:
I simply renamed this attribute to make it private (I don't think we need to have it public).